Button/tests/Button-specs.js

```diff
@@ -198,6 +198,17 @@ describe('Button', () => {
 		});
 	});
 
+	describe('with shadowed', () => {
+		test('should have shadowed class', () => {
```

Comment:
Would you please add for a test that `should not have shadowed class when the background is not 'transparent``?
```suggestion
		test('should have shadowed class when the background is transaprent', () => {
```